src/pages/index.md

```diff
@@ -10,18 +10,28 @@ If you would like to contribute to the docs, read [Contributing to the Docs][con
 
 <div class="container">
     <div class="row">
-        <div class="col col--6" style={{padding: '10px'}}>
+        <div class="col" style={{padding: '10px'}}>
             <Card
                 title="NeoForge Documentation"
                 body="Learn how to create your first mod using NeoForge, and discover the vast APIs it provides."
                 link="/docs/gettingstarted/"
                 linkTitle="Get Started"
             />
         </div>
+    </div>
+    <div class="row">
+        <div class="col col--6" style={{padding: '10px'}}>
+            <Card
+                title="ModDevGradle Documentation"
+                body="Learn about the Gradle plugin empowering developers to create mods for NeoForge."
+                link="/moddevgradle/docs/"
+                linkTitle="Read More"
+            />
+        </div>
         <div class="col col--6" style={{padding: '10px'}}>
             <Card
                 title="NeoGradle Documentation"
-                body="Learn about the Gradle plugin empowering developers to create mods for NeoForge and develop NeoForge itself."
+                body="Learn about the Gradle plugin used to develop NeoForge itself."
```

Comment:
NG is no longer used in neodev.

Reply:
How would you describe NG then? I couldn't think of a text beside MDG put with more advanced features for specific needs.

Reply:
How would you describe NG then? I couldn't think of a text beside MDG put with more advanced features for specific needs.

Reply:
Maybe call them "simple" and "advanced" Gradle plugins, respectively?